### PR TITLE
Add track GC to View methods new.

### PR DESCRIPTION
**Pull request by iemelyanov**

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -1,5 +1,10 @@
 .. _changes:
 
+4.5.3 (2018-12-16)
```
Comment (asvetlov) [inline docs/changes.rst]: Please leave me the privilege to decide when the next release will be published.

XXX for a new version and date would be fine as placeholders.

Comment (iemelyanov) [reply docs/changes.rst]: Oh, sorry, I misunderstand you and did more than needed.

File: docs/changes.rst [hunk 2]
```diff
@@ -1,5 +1,10 @@
 .. _changes:
 
+XXX (2018-12-16)
+----------------
+
+* Fix memory leak introduced by 4.5.0 release (:pr:`314`)
```
[the remark this answers is not in the document]
Comment (iemelyanov) [reply docs/changes.rst]: Yes, you right. Copy-past at night... Mb "Fix GC object tracking."?

[the remark this answers is not in the document]
Comment (asvetlov) [reply docs/changes.rst]: ok

**Pull request merged**